### PR TITLE
Ground Items: Add notification by value config option

### PR DESCRIPTION
**Pull request by TheStonedTurtle**
Pretty much does the title

image

This also fixes a bug in the notification where it should have displayed `Lots` but was instead display 65535

image

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java [hunk 1]
```diff
@@ -234,15 +234,20 @@ public void onItemSpawned(ItemSpawned itemSpawned)
...
 			groundItem.getGePrice(),
-			groundItem.getHaPrice()));
+			groundItem.getHaPrice());
+		boolean shouldNotify = !config.onlyShowLoot() && config.highlightedColor().equals(highlightedColor);
 
 		if (config.notifyHighlightedDrops() && shouldNotify)
 		{
 			notifyHighlightedItem(groundItem);
 		}
+		else if (highlightedColor != null && (!config.onlyShowLoot() || groundItem.isMine()) && getGroundItemConfigValue(groundItem) > config.getNotifyOverValue())
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: Why does this require `highlightedColor != null`?

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java [hunk 2]
```diff
@@ -651,21 +670,15 @@ public void onFocusChanged(FocusChanged focusChanged)
...
-			.append(item.getName());
+		final StringBuilder notificationStringBuilder = new StringBuilder(item.getName());
 
 		if (item.getQuantity() > 1)
 		{
 			notificationStringBuilder.append(" x ").append(item.getQuantity());
 
-
-			if (item.getQuantity() > (int) Character.MAX_VALUE)
+			if (item.getQuantity() >= (int) Character.MAX_VALUE)
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: This should be in a separate commit.

File: runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java (hunk 1 above)
Comment (TheStonedTurtle) [reply runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java]: I added this to ensure that items that are explicitly hidden would not trigger valuable notifications, but now I see this would prevent notifications from working if their price checkpoints are lower than their notification threshold.

Should I still prevent items that are explicitly hidden from triggering this notification?
